src/index.js

```diff
@@ -3,7 +3,11 @@ import * as t from "@babel/types";
 
 // this is the code that we will generate for Pressability
 const _onFsPressForward_PressabilityCode = `_onFsPressForward_Pressability = function(isLongPress) {
-  if (!UIManager || !UIManager.onFsPressForward) {
+  try {
+    if (!UIManager || !UIManager.onFsPressForward) {
+      return;
+    }
+  } catch (e) {
```

Comment:
I think what we need is a `typeof` check here - that's allowed to operate on undefined variables:

```suggestion
  if (typeof UIManager === "undefined" || !UIManager || !UIManager.onFsPressForward) {
```

Reply:
Per slack conversation, this doesn't work - https://fullstory.slack.com/archives/C012TFHSY2Z/p1651675541148699?thread_ts=1651603235.694929&cid=C012TFHSY2Z